src/alphabets/dna.rs

```diff
@@ -34,7 +34,7 @@ pub fn n_alphabet() -> Alphabet {
 
 /// The IUPAC DNA alphabet (uppercase and lowercase).
 pub fn iupac_alphabet() -> Alphabet {
-    Alphabet::new(b"ACGTURYSWKMBDHVNacgturyswkmbdhvn")
+    Alphabet::new(b"ACGTRYSWKMBDHVNZacgtryswkmbdhvnz")
```

Comment:
I am unsure where this comes from (I would need to check it myself), but you are right that since we now have a distinction between RNA and DNA alphabets, there should be no Uracil in here. Hence, this is fine.